include/common.sh

```diff
@@ -316,8 +316,14 @@ ansiline()
 
 getScreenCols()
 {
-  # Note tput does not handle ssh sessions too well...
-  stty | awk -F'[/=;]' '/columns/ { print $4}' | tr -d " "
+  if  [ -t 1 ] && [ -t 2 ]; then
```

Comment:
I had to look this up - might just be me - but would it make sense to have a comment that says that -t fd tests if the descriptor is opened on a terminal?

Reply:
thanks, added a comment